racer.el

```diff
@@ -123,6 +124,10 @@
       (substring s (+ idx (length needle)))
     s))
 
+(defun racer-complete--prefix-p (beg end)
+  "Return if a completion should be triggered for a prefix between BEG and END."
```

Comment:
Oh, I see. I read "return if" as "terminate if" whereas you meant "return whether a completion should be triggered". Perhaps "return t if" is more explicit.

Admittedly the function is tiny and readable anyway.